app/policies/rubygem_policy.rb

```diff
@@ -0,0 +1,27 @@
+class RubygemPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.with_versions
```

Comment:
See e.g. https://rubygems.org/gems/hpriocot. We _do_ allow showing gems without versions, so I am not sure this is correct.